scripts/explain_manifest/fixtures/amazonlinux-2-amd64.txt

```diff
@@ -66,6 +66,16 @@
   - libc.so.6
   Runpath   : /usr/local/kong/lib
 
+- Path      : /usr/local/kong/modules/ngx_wasm_module.so
```

Comment:
@locao @gszr is `ngx_wasm_module.so` supposed to be reported twice in the manifest file?

Reply:
Yes, there are the actual shared object and the symbolic link to it. 

The shared object is installed in the default openresty path, but we decided to not rely on it. So, in `conf_loader` we use the symbolic link path (`/usr/local/kong/modules/ngx_wasm_module.so`) which is created in build time to wherever the module is installed.

Reply:
Oh gotcha.

I wonder if it's possible to just move the module into `{{ build prefix }}/kong/modules` instead of symlinking? I assume there's a docker/packaging/localdev reason the symlink was needed in the first place though, so maybe not :shrug:.

Anyhow, we probably have bigger fish to fry at the moment than thinking too hard about this :)